packages/components/components/elvis-popover/src/react/elvia-popover.tsx

```diff
@@ -93,6 +83,12 @@ export const Popover: FC<PopoverProps> = function ({
     updatePreferredPosition();
   }, [isShowingConnectedOverlayState, content, horizontalPosition, verticalPosition, heading]);
 
+  /* Add class to text-abbreviation element trigger element when the popover is open */
+  useEffect(() => {
+    const textAbbreviationElement = triggerRef.current?.querySelector('.e-text-abbreviation');
+    textAbbreviationElement?.classList.toggle('e-text-abbreviation---active', isShowingConnectedOverlayState);
+  }, [isShowingConnectedOverlayState]);
```

Comment:
This is not the nicest, but I prefer adding this little thing here over having to make our users make the popover controlled simply to add this one class when active 🤔

Reply:
Though it is something that won't be used very often, so I am ok with dropping this "nice-to-have" functionality if we would prefer that. Then our users will need to listen to `onOpen` and `onClose` to apply the active-state themselves.

Reply:
Maybe we should drop this then? Then it is the same as Context Menu and its "button pressed"-state:
![Skjermbilde 2024-04-25 kl  15 00 05](https://github.com/3lvia/designsystem/assets/54577949/ea8ada5a-c2c3-4c96-8ccc-22169e6b4743)
On the other hand, we will need to document `'e-text-abbreviation---active'`